src/api/restaurants.ts

```diff
@@ -0,0 +1,9 @@
+export async function getRestaurants() {
+  const res = await fetch('http://localhost:3000/restaurants/api');
```

Comment:
https://github.com/tperamaki/planning-poker/blob/main/src/network.ts#L5

tästä vois droppaa ton localhostin pois, ja pitäiskö polun olla ennemmin sillai et ekana ois /api ja sit jotain

```suggestion
  const res = await fetch('/api/restaurants');
```

Reply:
Ei toiminut lokaalissa ilman localhostia. Ihmettelin tota samaa eilen pienen tovin ja järkeilin, että jos fetch tehdään server componentsin puolella niin se vaatii ton etuliitteen 😕 Planning pokerissa tehdään ilmeisesti pelkästään clientissa kun käyttäjä painaa nappia?